src/classes/mech/Mech.ts

```diff
@@ -185,7 +185,7 @@ class Mech implements IActor {
   }
 
   public get IsActive(): boolean {
-    return this.Pilot.State.ActiveMech && this.Pilot.State.ActiveMech.ID === this.ID
+    return this.Pilot.State.ActiveMech?.ID === this.ID
```

Comment:
This wasn't really a necessary change, but I wanted to point out that Comp/Con's version of Typescript can support [Optional Chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining), which may improve readability of the code if implemented over time.

Reply:
I disagree with you about readability. The former is more verbose, but I believe that anyone who has experience with a declarative language can understand what is happening before.
Understanding what is happening now requires knowledge of what I would consider niche syntactical sugar for Typescript. ( I also have issues with how this syntactical sugar is presented. I'm not even sure how to search for what syntax would mean if you hadn't explained it. "typescript ? after object"?)

All that said, I am probably just being a curmudgeon who primarily works in C and bash scripting because of my day job. I do not intend to hold up these changes because of my opinion.